scripts/qgis_plugin_ci.py

```diff
@@ -127,6 +137,32 @@ def main():
 
     args = parser.parse_args()
 
+    # set log level depending on verbosity argument
+    args.verbosity = 40 - (10 * args.verbosity) if args.verbosity > 0 else 0
+    logging.basicConfig(
+        level=args.verbosity,
+        format="%(asctime)s||%(levelname)s||%(module)s||%(message)s",
+        datefmt="%Y-%m-%d %H:%M:%S",
+    )
+
+    console = logging.StreamHandler()
+    console.setLevel(args.verbosity)
+
+    # add the handler to the root logger
+    logger = logging.getLogger(__title_clean__)
+    logger.debug(f"Log level set: {logging}")
+
+    # print the version and exit
+    if args.version:
+        import pkg_resources
+
+        print(
+            "qgis-plugin-ci version: {}".format(
+                pkg_resources.get_distribution("qgis-plugin-ci").version
+            )
+        )
+        parser.exit()
```

Comment:
Left over from #153 when rebasing your PR ?

Reply:
Ups, it's all messed by a bad manip with my IDE. Sorry. I recreate a branch and a PR.